### PR TITLE
Deploy Static Nginx App to Prod

### PR DESCRIPTION
**Pull request by rickyp72**
Removed web app.
set NEXT_PUBLIC_API_URL to point at the ellandi frontend /api to allow Nginx proxy_paas to redirect api requests to the api app. 
Set the internal API usr for the api app.

### CODE REVIEW

File: api/ellandi/settings.py [hunk 1]
```diff
@@ -53,20 +53,14 @@
 
 HOST_MAP = {
     "http://testserver": "http://testserver",
-    "http://localhost:8000": "http://localhost:3000",
```
Comment (rottitime) [inline api/ellandi/settings.py:49]: @rickyp72 we still need to access localhost:8000 from 3000 for local development. This seems to break  CORs for local development

Screenshot 2022-10-05 at 12 07 40

**Review by rottitime — CHANGES_REQUESTED** [review]

**Review by rottitime — APPROVED** [review]

File: api/ellandi/settings.py (hunk 1 above)
Comment (rottitime) [reply api/ellandi/settings.py:49]: Thanks @rickyp72

**Comment by Singletoned** [comment]
There's no way this PR should have 143 commits in it

**Review by Singletoned — CHANGES_REQUESTED** [review]
This has way too many commits in it.  Something has gone wrong with your branching

**Comment by rickyp72** [comment]
@Singletoned This is a `feature branch` made directly from `develop`. These are just all the changes from `develop` that have not yet been `merged` to `main`

**Pull request closed without merging**